aspnet/mvc/models/file-uploads.rst

```diff
@@ -0,0 +1,197 @@
+File Uploads
+============
+
+By `Steve Smith`_
+
+ASP.NET MVC actions support uploading of one or more files, using simple model binding for smaller files, or streaming for larger files.
```

Comment:
We can, but I'm not sure what it should be. I would probably say anything less than 10MB is "small" but from my understanding it can become an issue if there is very frequent uploading of smaller files just as much as less frequent uploading of larger files (example: 20 simultaneous 2MB uploads vs. 2 simultaneous 20MB uploads). Perhaps the product team can weigh in?